docs/playground/ko/3-7/Fixits/Infer From Usage Changes.ts

```diff
@@ -0,0 +1,37 @@
+//// { compiler: {  noImplicitAny: false }, order: 2 }
+
+// TypeScript 3.7 버전에 있는 '사용 빈도 수에 따른(infer from usage)' 코드 수정은 
```

Comment:
```suggestion
// TypeScript 3.7 버전에 있는 '사용법에서 추론한' 코드 수정은 
```

글의 아래 내용을 보니 "사용 빈도수에 따른" 보다는 "사용법에서 추론한" 이 더 적합할 것 같은데 어떻게 생각하시나요 ??

arr.push(12); 라는 구문에서 arr 을 number[] 로 추론하는 기능에 대한 설명 같아서요 :)

```ts
function pushNumber(arr) { // arr 타입을 지정하지 않았지만 arr 을 number[] 로 유추함
  arr.push(12);
}
```

Reply:
네! 수정하도록 하겠습니다. 감사합니다.